### DISCUSSION TITLE
filerecursivebrowser is not working

### DISCUSSION THREAD

**Discussion by victorballester7** [davatorium/rofi#2041] [(Possible) Issues]
Hi, 
May I have some dependency installed in order to make the recursive file browser work? Because right now it doesn't show anything whenever I run it.
Any comment is appreciated!!! :)
Thanks!

**Comment by DaveDavenport** [comment]
No extra dependencies are required. What goes wrong is hard to say, it works for me. 

Is anything printed out if you run it on the commandline?

Can you paste output of rofi -v

**Comment by victorballester7** [comment]
Hi @DaveDavenport! Thanks for the answer! :)

Here is the output
```
rofi -v
Version: 1.7.5+wayland3
```
To be honest it used to work in the past, but not anymore after a clean reset of my computer. Should I need a file indexer program or rofi does it automatically?

**Reply by lbonn** [reply]
This is from the wayland fork and it does not have all the fixes in upstream yet.

Corresponding version is f2c0f75fd2faa656f0fe34df5029e0f4e6f445fc which is supposed to have a first version of the browser but not sure what the exact state is

**Reply by victorballester7** [reply]
Ok, I installed the _normal_ rofi `Version: 1.7.5` but still it tells me that recursive browser does not exist... Is it because is it a feature of the _next_ branch? If that's the case is there a way to easily updated if I initially installed the package through pacman (arch)?
Thanks !

**Reply by lbonn** [reply]
Yes it's an upcoming feature of rofi 1.7.6. The wayland branch and the 1.7.5+wayland releases are in an in-between state. As you are trying out some new features, I would in general use a git HEAD build

**Reply by victorballester7** [reply]
Ok, so I installed now `rofi-git` but still for some reason the recursive browser still do not respond when I run `rofi -show recursivebrowser`. I mean it opens rofi correctly, but if I start writing something, it seems like it doesn't look for files.
Here's my version now:
`Version: 1.7.5-249-g186b0e7b (makepkg)`
And my config file:
```
/*****----- Configuration -----*****/
configuration {
	modes:                       "combi,calc,drun,recursivebrowser";
  combi-modes: [drun, recursivebrowser];
  show-icons:                 true;
  display-drun:               " ";
  display-run:                " ";
  display-filebrowser:        " ";
  display-recursivebrowser:   " ";
  display-window:             " ";
  display-calc:               "󰪚 ";
  display-combi:             " ";
	font: "JetBrainsMono NF Extra-Bold 12";
	drun-display-format:        "{name}";
	window-format:              "{w}  ·  {c}  ·  {t}";
  combi-display-format: "{text}";
  kb-ellipsize: "Alt+e";
  sort: false;
  scroll-method: 1;
  me-select-entry: "MousePrimary";
  me-accept-entry: "!MousePrimary";
}
```
